packages/world/ts/actions/delegation.ts

```diff
@@ -0,0 +1,56 @@
+import type { WalletClient, Transport, Chain, Account, Hex, WalletActions, WriteContractReturnType } from "viem";
+import { getAction, getAbiItem, formatAbiItem, toFunctionSelector, type GetAbiItemParameters } from "viem/utils";
+import { writeContract } from "viem/actions";
+import { encodeSystemCallFrom, type SystemCallFrom } from "../encodeSystemCallFrom";
+
+type DelegationParameters = {
+  worldAddress: Hex;
+  delegatorAddress: Hex;
+  getSystemId: (functionSelector: Hex) => Hex;
+};
+
+// By extending clients with this function after delegation, the delegation automatically applies
+// to the World contract writes, meaning these calls are made on behalf of the delegator.
+export function delegation<TChain extends Chain, TAccount extends Account>({
```

Comment:
```suggestion
export function delegatedCalls<TChain extends Chain, TAccount extends Account>({
```
Since "delegation" is a bigger concept, how about calling this `delegatedCalls` or similar?

Reply:
The change makes sense!